server/static/static.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/version"
 	"google.golang.org/protobuf/encoding/protojson"
 
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/githubauth"
```

Comment:
```
          _____                    _____                _____          
         /\    \                  /\    \              /\    \         
        /::\____\                /::\    \            /::\    \        
       /::::|   |                \:::\    \           \:::\    \       
      /:::::|   |                 \:::\    \           \:::\    \      
     /::::::|   |                  \:::\    \           \:::\    \     
    /:::/|::|   |                   \:::\    \           \:::\    \    
   /:::/ |::|   |                   /::::\    \          /::::\    \   
  /:::/  |::|   | _____    ____    /::::::\    \        /::::::\    \  
 /:::/   |::|   |/\    \  /\   \  /:::/\:::\    \      /:::/\:::\    \ 
/:: /    |::|   /::\____\/::\   \/:::/  \:::\____\    /:::/  \:::\____\
\::/    /|::|  /:::/    /\:::\  /:::/    \::/    /   /:::/    \::/    /
 \/____/ |::| /:::/    /  \:::\/:::/    / \/____/   /:::/    / \/____/ 
         |::|/:::/    /    \::::::/    /           /:::/    /          
         |::::::/    /      \::::/____/           /:::/    /           
         |:::::/    /        \:::\    \           \::/    /            
         |::::/    /          \:::\    \           \/____/             
         /:::/    /            \:::\    \                              
        /:::/    /              \:::\____\                             
        \::/    /                \::/    /                             
         \/____/                  \/____/                              
                                                                       
```
can you move this up one line with the un-aliased imports

Reply:
Done - I blame vscode auto-import lol